.github/workflows/build.yml

```diff
@@ -37,10 +37,10 @@ jobs:
         echo 'Tag: ${{ steps.meta.outputs.tags }}'
         echo 'Label: ${{ steps.meta.labels.tags }}'
 
-#    - name: Build and push Docker image
-#      uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
-#      with:
-#       context: <go or python>
-#       push: true
-#       tags: ${{ steps.meta.outputs.tags }}
-#       labels: ${{ steps.meta.outputs.labels }}
\ No newline at end of file
+   - name: Build and push Docker image
+     uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+     with:
+      context: <go or python>
```

Comment:
which one?

Reply:
ahh i forgot to change that, ill edit it soon!